### PR TITLE
Cn translation

### PR DESCRIPTION
**Pull request by wuzhong-papermoon**
Translation for the following PR

https://github.com/moonbeam-foundation/moonbeam-docs/pull/761
https://github.com/moonbeam-foundation/moonbeam-docs/pull/720
https://github.com/moonbeam-foundation/moonbeam-docs/pull/756
https://github.com/moonbeam-foundation/moonbeam-docs/pull/755
https://github.com/moonbeam-foundation/moonbeam-docs/pull/749

### CODE REVIEW

File: builders/get-started/networks/moonbeam-dev.md [hunk 1]
```diff
@@ -62,9 +62,12 @@ Moonbeam开发节点是您自己的个人开发环境，用于在Moonbeam上构
         --dev --ws-external --rpc-external
         ```
 
-    如果节点已经启动，您将看到显示区块待创建的空闲状态界面：
+!!! note
```
Comment (eshaben) [inline builders/get-started/networks/moonbeam-dev.md:65]:
```suggestion
!!! 注意事项
```

File: builders/get-started/networks/moonbeam-dev.md [hunk 2]
```diff
@@ -92,6 +95,10 @@ purestake/moonbeam \
 
 2. 如果您已安装Rust，您可跳过以下两个步骤。如果您未安装Rust，请执行以下命令[通过Rust推荐方式](https://www.rust-lang.org/tools/install){target=_blank}安装Rust和其先决条件：
 
+!!! note
+        安装路径包含空格会造成编译错误
+
```
Comment (eshaben) [inline builders/get-started/networks/moonbeam-dev.md:100]:
```suggestion
!!! 注意事项
    安装路径包含空格会造成编译错误
```

**Review by eshaben — CHANGES_REQUESTED** [review]
Nice job! Just a couple minor things! 🙂 

For posterity's sake, just noting this one was not in the description, but was translated: https://github.com/moonbeam-foundation/moonbeam-docs/pull/749/files

**Comment by eshaben** [comment]
Also just noticed the merge conflicts, can you please resolve them @wuzhong-papermoon?

File: builders/build/substrate-api/polkadot-js-api.md [hunk 3]
```diff
@@ -268,7 +311,31 @@ api.disconnect();
...
+const tx = api.tx.balances.transfer('INSERT_BOBS_ADDRESS', BigInt(12345));
+
+// Get weight info
+const { partialFee, weight } = await tx.paymentInfo('INSERT_SENDERS_ADDRESS');
+
+console.log(`Transaction weight: ${weight}`);
+console.log(`Transaction fee: ${partialFee.toHuman()}`);
+```
+
+??? code "View the complete script"
```
[the remark this answers is not in the document]
Comment (wuzhong-papermoon) [reply builders/build/substrate-api/polkadot-js-api.md:334]:
```suggestion
??? code "查看完整文件"
```

**Comment by wuzhong-papermoon** [comment]
Hi Erin! 
Thanks for taking the time to review this PR. All the suggested changes have been made.

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**